### PR TITLE
[CARBONDATA-1475] fix default maven dependencies for Intellij Idea

### PR DESCRIPTION
**Pull request by QiangCai**
When we choose spark-1.6, Intellij Idea dependency scala-2.11.
This PR will fix this issue.
selection_007

### CODE REVIEW

File: examples/spark2/pom.xml [hunk 1]
```diff
@@ -38,32 +38,6 @@
       <groupId>org.apache.carbondata</groupId>
       <artifactId>carbondata-spark2</artifactId>
       <version>${project.version}</version>
-      <exclusions>
-        <exclusion>
-          <groupId>org.apache.spark</groupId>
```
Comment (chenliang613) [inline examples/spark2/pom.xml]: why need remove these exclusions ?  in spark2 module, need to exclude 2.10 dependency jars

**Comment by ravipesala** [comment]
SDV Build Fail , Please check CI http://144.76.159.231:8080/job/ApacheSDVTests/710/

**Comment by QiangCai** [comment]
please retest this

**Comment by CarbonDataQA** [comment]
Build Failed  with Spark 2.1.0, Please check CI http://136.243.101.176:8080/job/ApacheCarbonPRBuilder1/2/

File: examples/spark2/pom.xml (hunk 1 above)
Comment (QiangCai) [reply examples/spark2/pom.xml]: now spark-2.1 is the default profile. no need to exclusions spark-2.1 dependencies in spark2 module.

Comment (QiangCai) [reply examples/spark2/pom.xml]: I moved these exclusions to spark module.

**Comment by ravipesala** [comment]
SDV Build Fail , Please check CI http://144.76.159.231:8080/job/ApacheSDVTests/718/

**Comment by ravipesala** [comment]
SDV Build Fail , Please check CI http://144.76.159.231:8080/job/ApacheSDVTests/719/

**Comment by chenliang613** [comment]
retest this please

**Comment by ravipesala** [comment]
Build Success with Spark 1.6, Please check CI http://144.76.159.231:8080/job/ApacheCarbonPRBuilder/830/

**Comment by ravipesala** [comment]
SDV Build Fail , Please check CI http://144.76.159.231:8080/job/ApacheSDVTests/726/

**Comment by QiangCai** [comment]
mvn clean test -Pspark-2.1
[INFO] ------------------------------------------------------------------------
[INFO] Reactor Summary:
[INFO] 
[INFO] Apache CarbonData :: Parent ........................ SUCCESS [  1.689 s]
[INFO] Apache CarbonData :: Common ........................ SUCCESS [  7.008 s]
[INFO] Apache CarbonData :: Core .......................... SUCCESS [01:12 min]
[INFO] Apache CarbonData :: Processing .................... SUCCESS [ 21.645 s]
[INFO] Apache CarbonData :: Hadoop ........................ SUCCESS [ 19.416 s]
[INFO] Apache CarbonData :: Spark Common .................. SUCCESS [ 47.740 s]
[INFO] Apache CarbonData :: Spark2 ........................ SUCCESS [03:49 min]
[INFO] Apache CarbonData :: Spark Common Test ............. SUCCESS [11:23 min]
[INFO] Apache CarbonData :: Assembly ...................... SUCCESS [  3.083 s]
[INFO] Apache CarbonData :: Hive .......................... SUCCESS [ 20.557 s]
[INFO] Apache CarbonData :: presto ........................ SUCCESS [ 16.347 s]
[INFO] Apache CarbonData :: Spark2 Examples ............... SUCCESS [ 15.328 s]
[INFO] ------------------------------------------------------------------------
[INFO] BUILD SUCCESS
[INFO] ------------------------------------------------------------------------
[INFO] Total time: 18:58 min
[INFO] Finished at: 2017-09-13T20:00:05+08:00
[INFO] Final Memory: 135M/842M
[INFO] ------------------------------------------------------------------------

 mvn clean test -Pspark-1.6
[INFO] ------------------------------------------------------------------------
[INFO] Reactor Summary:
[INFO] 
[INFO] Apache CarbonData :: Parent ........................ SUCCESS [  2.756 s]
[INFO] Apache CarbonData :: Common ........................ SUCCESS [  7.896 s]
[INFO] Apache CarbonData :: Core .......................... SUCCESS [01:17 min]
[INFO] Apache CarbonData :: Processing .................... SUCCESS [ 22.714 s]
[INFO] Apache CarbonData :: Hadoop ........................ SUCCESS [ 19.481 s]
[INFO] Apache CarbonData :: Spark Common .................. SUCCESS [ 49.917 s]
[INFO] Apache CarbonData :: Spark ......................... SUCCESS [02:11 min]
[INFO] Apache CarbonData :: Spark Common Test ............. SUCCESS [10:46 min]
[INFO] Apache CarbonData :: Assembly ...................... SUCCESS [  0.478 s]
[INFO] Apache CarbonData :: Spark Examples ................ SUCCESS [ 17.637 s]
[INFO] Apache CarbonData :: Flink Examples ................ SUCCESS [  1.704 s]
[INFO] ------------------------------------------------------------------------
[INFO] BUILD SUCCESS
[INFO] ------------------------------------------------------------------------
[INFO] Total time: 16:18 min
[INFO] Finished at: 2017-09-13T20:38:30+08:00
[INFO] Final Memory: 97M/769M
[INFO] ------------------------------------------------------------------------

**Comment by QiangCai** [comment]
please retest this

**Comment by CarbonDataQA** [comment]
Build Success with Spark 1.6, Please check CI http://88.99.58.216:8080/job/ApacheCarbonPRBuilder/144/

**Comment by CarbonDataQA** [comment]
Build Success with Spark 2.1.0, Please check CI http://136.243.101.176:8080/job/ApacheCarbonPRBuilder1/268/

**Comment by ravipesala** [comment]
SDV Build Fail , Please check CI http://144.76.159.231:8080/job/ApacheSDVTests/900/

**Comment by ravipesala** [comment]
SDV Build Fail , Please check CI http://144.76.159.231:8080/job/ApacheSDVTests/1198/

**Comment by CarbonDataQA** [comment]
Build Success with Spark 2.1.0, Please check CI http://136.243.101.176:8080/job/ApacheCarbonPRBuilder1/567/

**Comment by QiangCai** [comment]
close this pr. not required

**Pull request closed without merging**